src/cure.erl

```diff
@@ -26,6 +26,9 @@
 %% Description and complete License: see LICENSE file.
 %% -------------------------------------------------------------------
 
+%% @doc Provides a protocol to replicate updates from one cluster to another
+%%      in the form of a transaction API.
+
```

Comment:
Maybe give a short description of CURE, the functions it provides, why it needs fixed static sharding, and reference the paper

Reply:
Still missing the fixed sharding bit